docs/arf.md

```diff
@@ -772,6 +758,23 @@ established by [Article 5a](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=
     (QES) Remote Service Provider. This interface is specifically used
     to execute a QES remote signature.
 
+Furthermore, the following interfaces are subject to harmonization, but are
+not considered as interoperable, standardized protocols:
+
+-   **The Wallet Provider Interface (WPI)** is used by the Wallet
+    Instance to communicate with the Wallet Provider to issue the Wallet
+    Trust Evidence and the Wallet Instance Attestation, as well as
+    provide support to the user and collect metrics for accounting
+    purposes.
+
+-   **The Secure Cryptographic Interface (SCI)** enables the Wallet
```

Comment:
I think a definition, at least, of the capabilities of this interface makes sense. It would allow to build wallet apps on top of different kinds of WSCDs.